src/math/Math.js

```diff
@@ -259,21 +259,28 @@ FORGE.Math.sphericalToCartesian = function(radius, theta, phi)
  * @param {number} x - x
  * @param {number} y - y
  * @param {number} z - z
+ * @param {string} [unit=radian] - The unit used to return spherical
  * @return {SphericalCoordinates}
  */
-FORGE.Math.cartesianToSpherical = function(x, y, z)
+FORGE.Math.cartesianToSpherical = function(x, y, z, unit)
 {
     var res = {};
 
     res.radius = Math.sqrt(x*x + y*y + z*z);
 
     if (res.radius === 0)
     {
-        return { radius: 0, theta: 0, phi: 0 }
+        return { radius: 0, theta: 0, phi: 0 };
     }
 
     res.phi = Math.asin(y / res.radius);
     res.theta = Math.atan2(x, -z || 0); // we want to avoid -z = -0
 
+    if(typeof unit === "string" && unit.toLowerCase().substring(0, 3) === "deg")
```

Comment:
BTW about this condition my first thought was to be able to accept "deg" "degree" or "degrees". I couldn't make a choice haha!